lib/bulk_insert/worker.rb

```diff
@@ -36,6 +36,10 @@ def initialize(connection, table_name, primary_key, column_names, set_size=500,
       @set = []
     end
 
+    def inserted_ids
+      @return_primary_keys ? @result_sets.map(&:rows).flatten : nil
+    end
```

Comment:
This option on non postgresql adapters may lead to some errors like:

Old rails versions

```
NoMethodError: undefined method `rows' for nil:NilClass
    /home/travis/build/jamis/bulk_insert/lib/bulk_insert/worker.rb:40:in `map'
    /home/travis/build/jamis/bulk_insert/lib/bulk_insert/worker.rb:40:in `inserted_ids'
    /home/travis/build/jamis/bulk_insert/test/bulk_insert/worker_test.rb:163:in `block in <class:BulkInsertWorkerTest>'
```

Or it may return empty arrays in SQLite and Mysql (only if rails >=6).

In order to avoid confusion and future bugs, I would suggest to fail as soon as possible in case of misconfiguration.

E.g. https://github.com/jamis/bulk_insert/blob/4a69affacf9714293710fb84e391bad14dca6ac3/lib/bulk_insert/worker.rb#L22-L23

```
# Add at line 23
validate_return_primary_keys!

# ---
def validate_return_primary_keys!
  if @return_primary_keys && !@statement_adapter.is_a?(StatementAdapters::PostgreSQLAdapter)
    raise ArgumentError, "Unsupported option 'return_primary_keys' for adapter #{@adapter_name}"
  end
end
```